install-script.sh

```diff
@@ -65,7 +65,9 @@ echo "OUTPUT FILE = ${OUTFILE}"
 cd "$SRCDIR"
 rm -rf .build
 case $(uname -s) in
-    Linux*)     swift build -c release
+    Linux*)     swift build -c release \
+                    -Xlinker -rpath -Xlinker /usr/lib/swift/linux \
+                    -Xlinker -rpath -Xlinker /usr/share/swift/usr/lib/swift/linux
```

Comment:
Binaries built with Swiftly set the rpath to Swiftly's toolchain, this has portability issues. So I using a standard rpath instead:
- /usr/lib/swift/linux
    The standard path used for Linux builds
- /usr/share/swift/usr/lib/swift/linux
    The path used by GitHub Actions